packages/calcite-components/src/components/dropdown/dropdown.tsx

```diff
@@ -634,29 +637,34 @@ export class Dropdown
     }
   }
 
-  private focusOnFirstActiveOrFirstItem = (): void => {
-    this.getFocusableElement(
-      this.getTraversableItems().find((item) => item.selected) || this.items[0],
-    );
+  private focusOnFirstActiveOrDefaultItem = (focusLastItem: boolean): void => {
+    const selectedItem = this.getTraversableItems().find((item) => item.selected);
+    if (selectedItem) {
```

Comment:
For readability, can you add a newline before the `if` statement?

Reply:
This will be irrelevant after https://github.com/Esri/calcite-design-system/pull/10264#discussion_r1752922398